examples/chain.js

```diff
@@ -1,9 +1,9 @@
-var Nemo = require("../");
-var config = {
-  "driver": {
-    "browser": "phantomjs"
-  },
-  'data': {
+const Nemo = require('../');
+const config = {
+  'driver': {
+    'browser': 'phantomjs'
+  }
+  ,'data': {
```

Comment:
What is the rationale for making the change to a "leading" comma in some of these cases?